README.md

```diff
@@ -53,6 +53,17 @@ We welcome your contributions, please adhere to the following guidelines when co
 * Merges from `develop` to `master` are only done for code releases. This way we always have a clean `master` that reflects the current release
 * Code should be formatted using [black](https://pypi.org/project/black/) style
 
+## Testing
+* First, install the test requirements (if not already installed in the virtual env with `pipenv install --dev`):
+```sh
+$ pip install -r tests/requirements.txt
```

Comment:
```suggestion
# activate environment first (optional)
$ pipenv shell

# install atoMEC as editable project in current directory (for developers)
$ pip install -e .[tests]

# alternatively install package from PyPI with test dependencies
$ pip install atoMEC[tests]
```

Reply:
I'm wondering if we should streamline things a bit? Do we need both the `--dev` option for the `pipenv install` and a `tests/requirements.txt`? Should we just remove the `dev-packages` section of the `Pipfile` and use the above method only to install the test dependencies?

Reply:
No, no, no:) Yes, we need both. With a `requirements.txt` one specifies usually dependencies your package relies on with a minimum version number like `numpy>=1.20.3`.  This tells us that your package makes use of features of numpy only included in versions equal or higher than 1.20.3. The end-user is free to install the package in whatever (virtual) environment he likes (be it venv, virtualenv, pipenv, poetry,...). The install process will just fail if the specified requirements are not met. By providing a `Pipfile` especially a `Pipfile.lock` we additionally serve an environment for which we guarantee `atoMEC` to function properly. Same goes with `docs/requirements.txt` and `tests/requirements.txt`, though we do not specify minimum version numbers here (one can argue about this here). What is not so nice is, that a `Pipfile` only offers specifying core `[packages]` and dev `[dev-packages]` dependencies, no custom sections for optional dependencies (so instead of `[dev-packages]` for example `[test-packages]`). (This is possible with [Poetry](https://python-poetry.org/), one can also tell Poetry which Python version to use for the current project. So we we should consider switching from pipenv to Poetry at some point.)

Reply:
Ok, roger that!